src/Elliptic/Systems/Xcts/Tags.hpp

```diff
@@ -121,5 +125,112 @@ struct ShiftStrain : db::SimpleTag {
   using type = tnsr::ii<DataType, Dim, Frame>;
 };
 
+/*!
+ * \brief The conformal longitudinal operator applied to the shift excess
+ * \f$(\bar{L}\beta^\mathrm{excess})^{ij}\f$
+ *
+ * This quantity can be used as the "flux" for the momentum constraint in
+ * formulations of the XCTS equations, because the principal part of the
+ * momentum constraint is essentially the divergence of this quantity.
+ */
+template <typename DataType, size_t Dim, typename Frame>
+struct LongitudinalShiftExcess : db::SimpleTag {
+  using type = tnsr::II<DataType, Dim, Frame>;
+};
+
+/*!
+ * \brief The conformal longitudinal operator applied to the background shift
+ * vector minus the time derivative of the conformal metric
+ * \f$(\bar{L}\beta^\mathrm{background})^{ij} - \bar{u}^ij\f$
+ *
+ * Note that the time derivative of the conformal metric has its indices raised
+ * as \f$\bar{u}^ij = \bar{\gamma}^{ik}\bar{\gamma}^{jl}
+ * \partial_t\bar{\gamma}_{kl}\f$, i.e. by the conformal metric, as usual for
+ * conformal quantities.
+ *
+ * \see `Xcts::Tags::ShiftBackground`
+ */
+template <typename DataType, size_t Dim, typename Frame>
+struct LongitudinalShiftBackgroundMinusDtConformalMetric : db::SimpleTag {
+  using type = tnsr::II<DataType, Dim, Frame>;
+};
+
+/*!
+ * \brief The conformal longitudinal operator applied to the shift vector minus
+ * the time derivative of the conformal metric, squared:
+ * \f$\left((\bar{L}\beta)^{ij} - \bar{u}^ij\right)
+ * \left((\bar{L}\beta)_{ij} - \bar{u}_ij\right)\f$
+ */
+template <typename DataType>
+struct LongitudinalShiftMinusDtConformalMetricSquare : db::SimpleTag {
+  using type = Scalar<DataType>;
+};
+
+/*!
+ * \brief The conformal longitudinal operator applied to the shift vector minus
+ * the time derivative of the conformal metric, squared and divided by the
+ * square of the lapse:
+ * \f$\frac{1}{\alpha^2}\left((\bar{L}\beta)^{ij} - \bar{u}^ij\right)
+ * \left((\bar{L}\beta)_{ij} - \bar{u}_ij\right)\f$
+ */
+template <typename DataType>
+struct LongitudinalShiftMinusDtConformalMetricOverLapseSquare : db::SimpleTag {
+  using type = Scalar<DataType>;
+};
+
+/*!
+ * \brief The shift vector contracted with the gradient of the trace of the
```

Comment:
Please specify whether this is the full shift vector, or only the shift-correction